### PR TITLE
Docs: screenshot/pdf add note on path not existing

### PR DESCRIPTION
**Pull request by ebidel**

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -637,7 +637,7 @@ Page is guaranteed to have a main frame which persists during navigations.
 
 #### page.pdf(options)
 - `options` <[Object]> Options object which might have the following properties:
-  - `path` <[string]> The file path to save the PDF to. If `path` is a relative path, then it is resolved relative to [current working directory](https://nodejs.org/api/process.html#process_process_cwd). If no path is provided, the PDF won't be saved to the disk.
+  - `path` <[string]> The file path to save the PDF to. If `path` is a relative path, then it is resolved relative to [current working directory](https://nodejs.org/api/process.html#process_process_cwd). If no path is provided or it doesn't exist, the PDF won't be saved to the disk.
```
Comment (aslushnikov) [inline docs/api.md]: don't we throw if the folders don't exist?

Comment (ebidel) [reply docs/api.md]: We do. Do you want to be that specific?

Comment (adelalsaggaf1) [reply docs/api.md]: اكيد

Comment (aslushnikov) [reply docs/api.md]: I'd state error explicitly: we do that in multiple other places in the doc.

Comment (ebidel) [reply docs/api.md]: done PTAL

**Review by aslushnikov — APPROVED** [review]

File: docs/api.md [hunk 2]
```diff
@@ -637,7 +637,7 @@ Page is guaranteed to have a main frame which persists during navigations.
 
 #### page.pdf(options)
 - `options` <[Object]> Options object which might have the following properties:
-  - `path` <[string]> The file path to save the PDF to. If `path` is a relative path, then it is resolved relative to [current working directory](https://nodejs.org/api/process.html#process_process_cwd). If no path is provided, the PDF won't be saved to the disk.
+  - `path` <[string]> The file path to save the PDF to. If `path` is a relative path, then it is resolved relative to [current working directory](https://nodejs.org/api/process.html#process_process_cwd). If no path is provided, the PDF won't be saved to the disk. If the path does not exist, the method throws an error.
```
Comment (aslushnikov) [inline docs/api.md:640]: oh wait, this is confusing now: the path technically doesn't exist here:
```js
page.screenshot({path: 'screenshot.png'})
```
Why do we want to do anything about this statement? Was there a confusion somewhere on the bugtracker?

**Review by aslushnikov — CHANGES_REQUESTED** [review]
(requesting changes to override previous "approved")

File: docs/api.md (hunk 2 above)
Comment (ebidel) [reply docs/api.md:640]: From https://github.com/GoogleChrome/puppeteer/pull/798

**Comment by ebidel** [comment]
I'm not convinced we need this either.

**Pull request closed without merging**